### PR TITLE
Improved README

### PR DESCRIPTION
**Pull request by JonathanMathBoi**
Added description and run instructions to README to fulfill requirements.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,39 @@
 # 2023fa-420-Xterminators
+
+## 📝 Description
+This project is an implementation of the Spelling Bee game in Java 20, created
+as a requirements of the CSCI 420 course. It allows players to play this word
```
Comment (JJSnader) [inline README.md:5]: requirements -> requirement

**Review by JJSnader — DISMISSED** [review]
I think it looks good, besides one minor typo.

File: README.md (hunk 1 above)
Comment (JonathanMathBoi) [reply README.md:5]: I originally had "created to fulfill the requirements of". I'll fix that.

**Review by JJSnader — APPROVED** [review]
Looks good.

**Pull request merged**